pom.xml

```diff
@@ -45,7 +45,7 @@
 
     <properties>
         <quarkus.version>1.3.2.Final</quarkus.version>
-        <hazelcast.version>4.0</hazelcast.version>
+        <hazelcast.version>4.0.1</hazelcast.version>
```

Comment:
How does that work? Is it provided dependency or updating quarkus-extension pull in latest hazelcast version?

Reply:
It's a normal non-provided dependency - just like other Quarkus extensions do